hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/mapreduce/MapReduceBackupMergeJob.java

```diff
@@ -240,7 +240,7 @@ protected void copyMetaData(FileSystem fs, Path tmpBackupDir, Path backupDirPath
    * @throws IOException exception
    */
   protected void copyFile(FileSystem fs, Path p, Path newPath) throws IOException {
-    File f = File.createTempFile("data", "meta");
+    File f = File.createTempFile("data", "meta", new File(System.getProperty("java.io.tmpdir")));
```

Comment:
Why do we need to download the whole file?

Just create an InputStream and an OutputStream and then use IOUtils.copy?

Reply:
@Apache9 Not sure if I understood what you meant. Can you explain a bit more detail?

Reply:
Sure. I got your point. let me try to fix it.

Reply:
@Apache9 Addressed your concern.